src/js/pouch.js

```diff
@@ -25,42 +46,79 @@ gpii.pouch.init = function (that) {
     fs.writeFileSync(that.options.pouchConfigPath, JSON.stringify(that.options.pouchConfig, null, 2));
 
     var MemPouchDB = PouchDB.defaults({ db: memdown });
-
-    fluid.each(that.options.databases, function (dbConfig, key) {
-        var db = new MemPouchDB(key);
-        if (dbConfig.data) {
-            var data = require(dbConfig.data);
-            db.bulkDocs(data);
-        }
-    });
-
     that.expressPouchdb = expressPouchdb(MemPouchDB, { configPath: pouchConfigPath });
 
-    //that.expressPouchdb.couchLogger.setFile(logPath);
+    if (PouchDB.isBeingCleaned) {
```

Comment:
Yes, that was left over from a previous attempt at a similar approach.  I will replace that with the new mechanism as suggested.